packages/components/src/input-control/index.tsx

```diff
@@ -101,7 +99,7 @@ export function UnforwardedInputControl(
 					onKeyDown={ onKeyDown }
 					onValidate={ onValidate }
 					paddingInlineStart={ prefix ? space( 2 ) : undefined }
-					paddingInlineEnd={ suffix ? space( 2 ) : undefined }
+					paddingInlineEnd={ undefined }
```

Comment:
I'm not sure the InputControl should be chaned. This is a base component that can be reused of other purposes and by other components.